Resources/views/Collector/db.html.twig

```diff
@@ -263,12 +265,16 @@
                                 </div>
 
                                 <div id="formatted-query-{{ i }}-{{ loop.parent.loop.index }}" class="sql-runnable hidden">
-                                    {{ query.sql|doctrine_pretty_query }}
+                                    {% set formatted_sql = query.sql|doctrine_pretty_query %}
+                                    {{ formatted_sql|raw }}
```

Comment:
Also why do we need `|raw` here now?

Reply:
because it's saved to variable now, that's how it works